demo/markdownlint-browser.js

```diff
@@ -4601,7 +4601,8 @@ module.exports = [
 
 const { addError, escapeForRegExp, filterTokens, forEachInlineChild, forEachHeading, htmlElementRe } = __webpack_require__(/*! ../helpers */ "../helpers/helpers.js");
 // Regular expression for identifying HTML anchor names
-const identifierRe = /(?:id|name)\s*=\s*['"]?([^'"\s>]+)/iu;
+const identifierRe = /id\s*=\s*['"]?([^'"\s>]+)/iu;
```

Comment:
Can we call this "idRe" so it's consistent?

Reply:
Done.